### PR TITLE
add copy from option

### LINKED ISSUE #13: Add "Copy from" to copy settings between areas.
Allows the user to set up for example raid and instantly use that setup in e.g. Dungeons as well without having to manually go through all the spells again.

### PR DESCRIPTION
**Pull request by nyy23**
closes #13

### CODE REVIEW

File: Config.lua [hunk 1]
```diff
@@ -307,18 +310,53 @@ local spells = {
         return not profile.general.area[info[2]].enabled
     end,
     args = {
+        selectedZone = {
+            name = L["Copy Settings From:"],
+            desc = L["Select the zone you want to copy settings from"],
+            order = 1,
+            type = "select",
+            values = function(info)
+                local tmpzones = {}
```
Comment (Nitrak) [inline Config.lua:319]: I think we should add an "empty" value here, so it is possible to go back to not having the "Copy" button clickable again after having selected something and choosing to not copy anything.

File: Config.lua [hunk 2]
```diff
@@ -307,18 +310,53 @@ local spells = {
         return not profile.general.area[info[2]].enabled
     end,
     args = {
+        selectedZone = {
```
Comment (Nitrak) [inline Config.lua:313]: We have used "Area" previously, we should be consistent. Both Area and Zone correctly convey the message, we just need to pick one and stay with it.

File: Config.lua (hunk 1 above)
Comment (nyy23) [reply Config.lua]: +1

File: Config.lua (hunk 2 above)
Comment (nyy23) [reply Config.lua:313]: renamed to area

**Review by Nitrak — APPROVED** [review]

**Pull request merged**